Taylor/UI/UIImage.swift

```diff
@@ -74,4 +74,37 @@ extension UIImage
         
         return tintedImage
     }
+
+    /**
+     Resizes a new instance of an image resized with the specified width and height ratio
+
+     - parameter widthRatio: The maximum desired image size in Bytes
+     - returns: New instance of UIImage
+     */
+    public func resize(maxSize : CGFloat) -> UIImage? {
```

Comment:
- Comment not in line with impl
- Variable named size but is CGFloat??
- Should not try to convert as PNG, it might fail

Reply:
So converting to a JPEG never fails ?

Reply:
No it could fail as well, you shouldn’t need to convert the image to Data anyway. If it’s juste to know the size, use UIImage.size.